ports/stm32/mboot/mboot.h

```diff
@@ -46,9 +46,24 @@
 #define MBOOT_ADDRESS_SPACE_64BIT (0)
 #endif
 
+typedef enum {
```

Comment:
It would be helpful to document what the allowed arguments for each state are here. (And say "pass 0" for ones that don't use the arg.)

Reply:
done